src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.html

```diff
@@ -56,10 +56,26 @@ <h5 [ngClass]="{ h5MarginTop: isEditing }" class="profile-contacts-h5-style">{{
             </div>
             <div *ngIf="isEditing">
               <div class="wrapper-btn">
-                <button aria-label="Add alternative email" class="btn-alternative-email"></button>
+                <button aria-label="Add alternative email" class="btn-alternative-email" (click)="toggleAlternativeEmail()"></button>
                 <p>{{ 'ubs-client-profile.btn.alternative-email' | translate }}</p>
```

Comment:
Here would be good have something like this-
p *ngIf="!alternativeEmailDisplay">{{ 'ubs-client-profile.btn.alternative-email' | translate }}</p
p *ngIf="alternativeEmailDisplay">remove alternative email</p....
or other check ))